### ISSUE TITLE
Non admin privileges/installation required

### ISSUE THREAD

**Issue by agsola** [srevinsaju/guiscrcpy#9]
I'm currently using scrcpy in a computer where I have no admin rights (Windows 10). 
Since it is just an executable, it works perfectly. 

However, guiscrcpy requieres installation and admin privileges. 

Could you distribute a "portable" version with an executable that doesn't require installation?

**Comment by srevinsaju** [comment]
@agsola, create portable file. You may now create the shortcut on your own. To update scrcpy executable, if v 1.11 is released, you may extract scrcpy Windows 32 bit zip file to the guiscrcpy directory. Released in Released v1.10.1. please verify and comment.

Issues shall be closed in 2 days

**Comment by agsola** [comment]
It works, thank you!

I've just found three (some Windows related).

1.- When pressing "fullscreen", I get "wmctrl" is not recognized.
2.- When reorienting, it seems to use linux routes:
```
LOG: Passing REORIENT [LANDSCAPE]
/system/bin/sh: adb: not found
LOG: Passing REORIENT [POTRAIT]
/system/bin/sh: adb: not found

```

3.- When trying to move the guiscrcpy bars for the first time not using the dots, it dumps this error and closes.
```

LOG: MainUI loaded
LOG: Options received by class are :  8000 None False True False
SCRCPY RUNNING
Traceback (most recent call last):
  File "E:\Srevin\Python\guiscrcpy\__main__.py", line 690, in mouseMoveEvent
AttributeError: 'MyAppv' object has no attribute 'oldPos'
```

If, instead, I first move the bar with the three dots, and then move it elsewhere, it just works. 
Maybe you forgot to initialize oldPos or properly check it is not null.

**Comment by srevinsaju** [comment]
@agsola , your bugs are now fixed, new security release for Windows 1.10.1.1 is released fixing all the bugs mentioned here. You may download the latest Release on the Releases page, Windows column, link modified..

1. Fullscreen is currently under development only. You can use Scrcpy shortcut `Ctrl + F` for Fullscreen, but for the button, you need to install additional sources like `PyAutoGui`. As this is not meant for power users, I left it for developers to uncomment the lines and use Fullscreen button. Now a message is given, when the fullscreen button is pressed, so that it is known to all users.

Using PyAutoGui had some problems initially, @rom1v pointed out. But now it seems fine, but I am personally looking forward for alternative solutions! If you are interested you may give a pull request

2.  Second one was a mistake on my part and I don't use Windows much so.
The error is in bash, you can separate commands with `;` (semicolon) and on Windows, it is colon.
So separating this, fixes the problem on both OSes. Linux currently has o problems

3.  I tried a workaround (even, I fell into this same error) for declaring the `oldPos` variable, so that `NameError` or `AttributeError` does not arise, but I don't know how far it works, Lets hope for the best.

Hope all of them works fine! Thanks for your concern in improving `guiscrcpy`

> You may either close your issue (if fixed) else your issue will be closed in 48 hours.
